### PR TITLE
feat(quote): add smartSlippage property to quote

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

- Add quote `v1.1.0` with ~`smartQuote`~ `smartSlippage` boolean property
- Add appData `v1.2.0` using the new quote version

# Testing

Unit tests

### CODE REVIEW

**Comment by fleupold** [comment]
Can we call it smartSlipage or autoSlippage instead of quote?

File: src/schemas/v1.2.0.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+  "$id": "https://cowswap.exchange/schemas/app-data/v1.2.0.json",
```
Comment (shoom3301) [inline src/schemas/v1.2.0.json:2]: Should we use ens domain here? (Not now, maybe in the future)

**Review by shoom3301 — APPROVED** [review]
Technically all good

File: src/schemas/v1.2.0.json (hunk 1 above)
Comment (alfetopito) [reply src/schemas/v1.2.0.json:2]: Good catch! It still uses the domain from the before times. As you say, not a must as this point, but we should.

**Comment by alfetopito** [comment]
> Can we call it smartSlipage or autoSlippage instead of quote?

We sure can, will update.

**Review by anxolin — COMMENTED** [review]
I hate a choosed that name (smart slippage), no its sticky :P 

Probably in the app data, would be a better name "suggested slippage" or sth, but I don't think strong enough to change it now

**Pull request merged**